Documentation/git-for-each-repo.txt

```diff
@@ -42,6 +42,15 @@ These config values are loaded from system, global, and local Git config,
 as available. If `git for-each-repo` is run in a directory that is not a
 Git repository, then only the system and global config is used.
 
+--keep-going::
```

Comment:
You may also want to update the synopsis at the top of the page.

Reply:
Hmm. I'll think about that. I did not want to add slightly uninteresting information there. Many Git commands merely list `<options>` there, to avoid having a really long (and hence less than useful) synopsis.